test/smoke/src/sql/areas/notebook/notebookView.test.ts

```diff
@@ -9,21 +9,24 @@ import * as minimist from 'minimist';
 import { afterSuite, beforeSuite } from '../../../utils';
 
 export function setup(opts: minimist.ParsedArgs) {
-	describe('NotebookView', () => {
+	describe('NotebookTreeView', () => {
 		beforeSuite(opts);
 		afterSuite(opts);
 
 		it('Pin a notebook', async function () {
 			const app = this.app as Application;
 			await app.workbench.sqlNotebook.view.focusNotebooksView();
-			await app.workbench.sqlNotebook.view.pinNotebook();
-			await app.workbench.sqlNotebook.view.waitForPinnedNotebookView();
+			const notebookIds = await app.workbench.sqlNotebook.view.getNotebookTreeItemIds();
+			// Pinning SQL notebook to prevent the Configure Python Wizard from showing, since Python is no longer set up when the NotebookTreeView test suite starts
```

Comment:
Picking the second item and hoping that it's a SQL notebook isn't great, can you instead have it actually select the Notebook by name so we can make sure that we're always pinning the specific Notebook we're looking for? If the order changes currently then this could easily break again.